pkg/types/tuf/v0.0.1/tuf_v0_0_1_schema.json

```diff
@@ -0,0 +1,30 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "http://rekor.sigstore.dev/types/tuf/tuf_v0_0_1_schema.json",
+    "title": "tuf v0.0.1 Schema",
+    "description": "Schema for tuf metadata entries",
+    "type": "object",
+    "properties": {
+        "version": {
+            "description": "TUF specification version",
+            "type": "string",
+            "enum": ["1.0.0"]
+        },
+        "manifest": {
+            "description": "tuf manifest",
+            "type": "object",
+            "$ref": "tuf_manifest_v0_0_1_schema.json"
+        },
+        "root" : {
+            "description": "root manifest containing about the public keys used to sign the manifest",
```

Comment:
Maybe we should call it "metadata" instead of "manifest" to be consistent with TUF terminology? Might make for easier reading.

Reply:
changed uniformly